Framework/PythonInterface/mantid/_plugins/__init__.py

```diff
@@ -25,7 +25,7 @@
 ###############################################################################
 # Load the C++ library and register the C++ class exports
 ###############################################################################
-from ..kernel import _shared_cextension
+from mantid.utils import _shared_cextension, import_mantid
 
 with _shared_cextension():
```

Comment:
I'm on the fence with this but do you think it is worth having `import_mantid` be named `import_mantid_cext` and then have the `with _shared_cextension():` inside that to avoid duplicating it?

Reply:
Good suggestion, currently every call is a `with _shared_cextension` anyway